packages/appsync-emulator-serverless/lambdaRunnerPython.js

```diff
@@ -0,0 +1,69 @@
+const log = require('logdown')('appsync-emulator:lambdaRunner');
+
+const parseErrorStack = error =>
+  error.stack
+    .replace(/at /g, '')
+    .split('\n    ')
+    .slice(1);
+
+const sendOutput = output => {
+  process.send({ type: 'success', output }, process.exit);
+};
+const sendErr = err => {
+  let error;
+  if (err instanceof Error) {
+    error = {
+      stackTrace: parseErrorStack(err),
+      errorType: err.constructor.name,
+      errorMessage: err.message,
+    };
+  } else {
+    error = err;
+  }
+  process.send({ type: 'error', error }, process.exit);
+};
+
+process.once(
+  'message',
+  async ({ serverlessDirectory, handlerPath, handlerMethod, payload}) => {
+    try {
+        log.info('load', module);
+   
+        var spawn = require('child_process').spawn          
+        var args = ["invoke", "local", "-f", handlerMethod]
+
+        var sls = spawn("sls", 
+            args, 
+            {
+                env: process.env,
+                stdio: ['pipe', null, 'pipe'],
+                shell: '/bin/bash',
+                cwd: serverlessDirectory
+            });
+
+        sls.stdin.write(JSON.stringify(payload) + "\n");
+        sls.stdin.end();
+        
+        let results = ''
+        sls.stdout.on('data', function(data) {
+            results = data.toString('utf8');
```

Comment:
I think we want something like:

```
results += data.toString('utf8').replace('\n','')
```

Reply:
This was a problem that I have in conflict, because the stdout of the `spawn` method. All the output returned by the lambda is redirected to stdout (inclusively logs), I had difficult to get the correct output, which in this case will be the last to be outputted. Not a big solution, more an hack.

Reply:
ahhh, interesting.  Since this emulator is meant for local development, I'm okay with this hack.  Please add some comments so that future readers will understand.